package.json

```diff
@@ -73,6 +73,7 @@
     "eslint-plugin-react-hooks": "^2.5.0",
     "git-branch-is": "^4.0.0",
     "husky": "^4.2.3",
+    "jest": "^29.7.0",
```

Comment:
Let remove the inclusion of Jest here. This project [uses Vitest](https://github.com/jeremyckahn/farmhand/blob/6419076acb608c99baf357e8b0851ee4ae7eb1b0/package.json#L93), so this package isn't necessary.

Reply:
It looks like this still needs to be reverted out of this PR.